webserver/views/api.py

```diff
@@ -253,7 +253,17 @@ def _validate_listen(listen):
                 if len(tag) > MAX_TAG_SIZE:
                     _log_raise_400("JSON document may not contain track_metadata.additional_info.tags "
                                    "longer than %d characters." % MAX_TAG_SIZE, listen)
-
+        # MBIDs
+        lmbid = listen['track_metadata']['additional_info'].get('release_mbid')
+        if not is_valid_uuid(lmbid):
+                _log_raise_400("Release MBID format invalid.")
+        cmbid = listen['track_metadata']['additional_info'].get('recording_mbid')
+        if not is_valid_uuid(cmbid):
+                _log_raise_400("Recording MBID format invalid.")
+        ambids = listen['track_metadata']['additional_info'].get('artist_mbids')
+        for ambid in ambids:
```

Comment:
If `artist_mbids` isn't in additional_info then .get will return None, use `.get('artist_mbids', [])` here